packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -335,7 +379,8 @@ class ApplicationCommand extends Base {
       // Future proof for options being nullable
       // TODO: remove ?? 0 on each when nullable
       (command.options?.length ?? 0) !== (this.options?.length ?? 0) ||
-      (command.defaultPermission ?? command.default_permission ?? true) !== this.defaultPermission ||
+      defaultMemberPermissions !== (this.defaultMemberPermissions ? this.defaultMemberPermissions.bitfield : null) ||
```

Comment:
Can't you use optional accessing here?

Reply:
oops yeah, was thinking I was working on the api one still which had to be converted.